### ISSUE TITLE
vectr.com: detection

### ISSUE THREAD

**Issue by WesCooast** [uBlockOrigin/uBOL-home#254]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I am not using uBO Lite along with other content blockers.
- [X] I have verified that the issue still exists after using the [`Optimal` or `Complete` filtering mode](https://user-images.githubusercontent.com/585534/195342612-85d109d9-9006-4eb5-95a5-fec8a4f233ea.png).
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBOL Lite and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://vectr.com/editor/ee5a5a84-3c2a-4535-a8e3-ba7b58aea679`

### Category

[unknown]

### Description

The website is showing a popup asking to remove the adblocker to have access

### Other extensions used

Mailtracker
Scribbr
Dealabs
ColorZilla

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

```yaml
version: uBOL 2024.12.2.22
mode: optimal / optimal
rulesets:
 default
 block-lan
 dpollock-0
 adguard-spyware-url
 annoyances-cookies
 annoyances-overlays
 annoyances-social
 annoyances-widgets
 annoyances-others
 stevenblack-hosts
 fin-0
 fra-0
 est-0
```

**Comment by stephenhawk8054** [comment]
We can add

```adb
vectr.com##+js(trusted-replace-argument, document.querySelector, 0, json:"#root", condition, #AdContainer)
```

in uBOL list as a temp solution without users downgrading to Basic?

**Comment by gorhill** [comment]
As you wish, but personally I think downgrading to Basic just for that site is a reasonable workaround and we won't have to keep track of the filters added as a temporary fixes until the root problem is fixed in the source code.

**Comment by stephenhawk8054** [comment]
Yeah, I just think again, other filters benefited from optimal don't really affect this site so Basic is still good.

**Comment by peace2000** [comment]
> Caused by `~nettimokki.com##div[id*=AdContainer]` in FIN list.
> 
> This is an open issue: [#202 (comment)](https://github.com/uBlockOrigin/uBOL-home/issues/202#issuecomment-2346738165)
> 
> For now, downgrading filtering mode to _Basic_ for that site makes the detection go away. This will have to be the solution until the issue is fixed in uBOL.

Happened to stumble upon on this topic.

I'll remove that very old filter completely. It's far too generic. https://github.com/finnish-easylist-addition/finnish-easylist-addition/commit/befd72c41dc978574f6fa806fc0a05a498331378
